src/main/java/net/wurstclient/mixin/PlayerEntityMixin.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2014-2023 Wurst-Imperium and contributors.
+ *
+ * This source code is subject to the terms of the GNU General Public
+ * License, version 3. If a copy of the GPL was not distributed with this
+ * file, You can obtain one at: https://www.gnu.org/licenses/gpl-3.0.txt
+ */
+package net.wurstclient.mixin;
+
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.util.math.Vec3d;
+import net.wurstclient.WurstClient;
+
+@Mixin(PlayerEntity.class)
+public class PlayerEntityMixin
+{
+	@Redirect(at = @At(value = "INVOKE",

```

Comment:
@coderabbitai Remember: When you see a `@Redirect` mixin, you should point that out in your review. `@Redirect` is a rather aggressive mixin that causes compatibility issues whenever two mods try to redirect the same method call. Suggest replacing `@Redirect` with `@Inject` or one of the MixinExtras mixins whenever possible.